plugin/suda.vim

```diff
@@ -6,3 +6,10 @@ let g:loaded_suda = 1
 if get(g:, 'suda_startup', 1)
   call suda#init()
 endif
+
+if get(g:, 'suda_smart_read')
+  augroup suda_smart_read
+    autocmd!
+    autocmd BufEnter * nested call suda#BufEnter()
+  augroup end
+endif
```

Comment:
may

Let's say it `suda_smart_edit` while it is to read and write.

Reply:
Agree.